Source/WebCore/rendering/style/StyleGradientImage.cpp

```diff
@@ -43,24 +43,21 @@
 
 namespace WebCore {
 
-static inline bool operator==(const StyleGradientImageStop& a, const StyleGradientImageStop& b)
-{
-    return a.color == b.color
-        && compareCSSValuePtr(a.position, b.position);
-}
-
-static bool stopsAreCacheable(const Vector<StyleGradientImage::Stop>& stops)
+template<typename Stops>
+static bool stopsAreCacheable(const Stops& stops)
 {
     for (auto& stop : stops) {
-        // FIXME: Do we need handle calc() here?
-        if (stop.position && stop.position->isFontRelativeLength())
-            return false;
         if (stop.color && stop.color->containsCurrentColor())
             return false;
```

Comment:
do you have plans for currentcolor?

Reply:
I believe currentColor works (I think you fixed it!). This is just about caching.

Reply:
Oh nice. I somehow thought gradients were still broken.